spec/spec_helper_acceptance.rb

```diff
@@ -1,66 +1,7 @@
 require 'voxpupuli/acceptance/spec_helper_acceptance'
 
+ENV['BEAKER_FACTER_GITLAB_IP'] = File.read(File.expand_path('~/GITLAB_IP')).chomp
```

Comment:
https://github.com/voxpupuli/voxpupuli-acceptance/#environment-variables-to-facts

I also thought about doing it in the workflow file but I think this is better for users who want to run it outside of GHA.